packages/gatsby/src/utils/worker/__tests__/queries.ts

```diff
@@ -331,4 +342,204 @@ describeWhenLMDB(`worker (queries)`, () => {
 
     spy.mockRestore()
   })
+
+  it(`should return actions occurred in worker to replay in the main process`, async () => {
+    const result = await worker.single.runQueries(queryIdsSmall)
+
+    expect(result).toMatchInlineSnapshot(`
```

Comment:
I like to not overuse (inline)snapshots so this here could possibly be refactored to a test that checks if the array of objects contains objects that have the shape of `payload` and `type`?

Because in the next test we test the exact shape again, this here could be a preliminary "does this actually return anything / something in the rough shape we expect" test

Reply:
Makes sense, will change 👍